src/utils/data.jl

```diff
@@ -54,6 +52,36 @@ function Base.download(
     return data
 end
 
+"""
+Download Data from a "branch" into a "data" folder in given argument path.
+Skip the actual download if the folder already exists and force=false.
+Returns the downloaded folder name.
+"""
+function Base.download(
+    repo::AbstractString,
+    branch::AbstractString,
+    folder::AbstractString,
+    force::Bool = false,
+)
+    if Sys.iswindows()
+        DATA_URL = "$repo/archive/$branch.zip"
+    else
+        DATA_URL = "$repo/archive/$branch.tar.gz"
+    end
+    directory = abspath(normpath(folder))
+    reponame = splitpath(repo)[end]
+    data = joinpath(directory, "$reponame-$branch")
+    if !isdir(data) || force
+        @info "Downloading $DATA_URL"
```

Comment:
If force is true and the directory exists then it would be a good idea to delete it first. The repo may have deleted a file.